### PR TITLE
done!

### PR DESCRIPTION
**Pull request by vadiq**

### CODE REVIEW

File: git/homework.py [hunk 1]
```diff
@@ -15,23 +15,32 @@ def is_two_object_has_same_value(first: Any, second: Any) -> bool:
     If @first and @second has same value should return True
     In another case should return False
     """
-    pass
+    if first == second:
+        return True
+    else:
+        return False
```
Comment (alchemixXx) [inline git/homework.py]: redundant statement "if". Operator "==" will already return True or False.
Make the refactoring.

File: git/homework.py [hunk 2]
```diff
@@ -15,23 +15,32 @@ def is_two_object_has_same_value(first: Any, second: Any) -> bool:
...
 def is_two_objects_has_same_type(first: Any, second: Any) -> bool:
     """
     If @first and @second has same type should return True
     In another case should return False
     """
-    pass
+    if type(first) == type(second):
+        return True
+    else:
+        return False
```
Comment (alchemixXx) [inline git/homework.py:29]: The same that in #1.

File: git/homework.py [hunk 3]
```diff
@@ -15,23 +15,32 @@ def is_two_object_has_same_value(first: Any, second: Any) -> bool:
...
 def is_two_objects_is_the_same_objects(first: Any, second: Any) -> bool:
     """
     If @first and @second has same type should return True
     In another case should return False
     """
-    pass
+    if id(first) == id(second):
+        return True
+    else:
+        return False
```
Comment (alchemixXx) [inline git/homework.py:37]: The same that in #1.

File: git/homework.py [hunk 4]
```diff
@@ -97,14 +112,23 @@ def is_word_in_text(word: str, text: str) -> bool:
         >>> False
 
     """
-    pass
+    if word in text:
+        return True
+    else:
+        return False
```
Comment (alchemixXx) [inline git/homework.py:109]: The same that in #1.

File: git/homework.py [hunk 5]
```diff
@@ -127,7 +157,13 @@ def alphabet() -> dict:
...
         >>> {"a": 1, "b": 2 ...}
     """
-    pass
+    abc = ('abcdefghijklmnopqrstuvwxyz')
+    alph = {}
+    j = 1
+    for i in abc:
+        alph[j] = i
+        j += 1
+    return alph
```
Comment (alchemixXx) [inline git/homework.py:156]: use python library alphabet instead line 160

**Review by alchemixXx — DISMISSED** [review]

File: git/homework.py (hunk 3 above)
Comment (dimigor) [reply git/homework.py:37]: use operator `is` instead

Comment (vadiq) [reply git/homework.py:37]: used is instead of id() but it's doubtful

Comment (dimigor) [reply git/homework.py:37]: Read about `is`)

**Review by dimigor — APPROVED** [review]

**Pull request merged**